packages/client/src/app/services/content/content.service.ts

```diff
@@ -26,16 +27,17 @@ export class ContentService {
       }));
   }
 
-  public fetchAllPublished(pageNum: number, kind: ContentKind, userId?: string) {
+  public fetchAllPublished(pageNum: number, kind: ContentKind[], userId?: string): Observable<PaginateResult<ContentModel>> {
     let route = `${this.url}`;
 
+    // If we just include the kind array as-is, it'll be serialized as "&kind=Kind1,Kind2" which the backend will interpret as
+    // the string 'Kind1,Kind2' which is not what we want. So, we manually split it out into a query string
+    const kindFragment = kind.map(k => `&kind=${k}`).join('');
```

Comment:
I did not expect NestJS to handle this as gracefully as it did. +1 for convenience here